web/app/controllers/application.ts

```diff
@@ -0,0 +1,16 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import ConfigService from "hermes/services/config";
+import config from "hermes/config/environment";
+
+export default class ApplicationController extends Controller {
+  @service("config") declare configSvc: ConfigService;
+
+  protected get animatedToolsAreShown() {
+    if (config.environment === "development") {
+      return this.configSvc.config.feature_flags["showEmberAnimatedTools"];
```

Comment:
I _think_ this should work along with the other suggestion I made below...
```suggestion
import config from "hermes/config/environment";

export default class ApplicationController extends Controller {
  protected get animatedToolsAreShown() {
    if (config.environment === "development") {
      return config.showEmberAnimatedTools;
```